exareme2/algorithms/exaflow/inputdata_utils.py

```diff
@@ -0,0 +1,145 @@
+from typing import Any
```

Comment:
The `fetch_data` should be the same between Flower and Exaflow.

The file should be located in the `algorithms.utils` folder.